rm_referee:
  robot_type: radar
  blue_receiver:
    -
